ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -499,7 +499,7 @@ withCommas ms =
 -- | Common build dependencies.
 commonBuildDepends :: [String]
 commonBuildDepends =
-  [ "ghc-prim > 0.2 && < 0.6"
+  [ "ghc-prim > 0.2 && < 0.6.1"
```

Comment:
Why `0.6.1` instead of `0.7`? I would expect ghc-prim to uphold pvp.

Reply:
I'm changing to 0.7 actually as we speak.